package.json

```diff
@@ -212,6 +217,8 @@
     "graphql-request": "^3.4.0",
     "immer": "^9.0.6",
     "inter-ui": "^3.13.1",
+    "jest": "^29.6.1",
```

Comment:
Can we use the same version of jest that is already installed, to avoid adding a new dependency?